chalice/app.py

```diff
@@ -266,15 +292,42 @@ def __init__(self, body, headers=None, status_code=200):
         self.headers = headers
         self.status_code = status_code
 
-    def to_dict(self):
+    def to_dict(self, binary_types=None):
```

Comment:
Is this method typically used by the end user? I wonder if it makes sense to expose a ``binary_types`` parameter as a user typically does not have to worry about specifying what is a binary type; they just need to specify the content type.